seqr/views/apis/individual_api.py

```diff
@@ -703,13 +703,18 @@ def _get_record_updates(record, individual, invalid_values, allowed_assigned_ana
             if k == ASSIGNED_ANALYST_COL:
                 if v not in allowed_assigned_analysts:
                     raise ValueError
-                parsed_val = v
+                if v:
```

Comment:
if v in false-y wouldn't we have raised the error for `v not in allowed_assigned_analysts`?